includes/Core/Admin_Bar/Admin_Bar.php

```diff
@@ -134,41 +134,55 @@ private function add_menu_button( $wp_admin_bar ) {
 	 * @return bool True if Admin bar should display, False when it's not.
 	 */
 	public function is_active() {
-		if ( ! is_user_logged_in() || ! is_admin_bar_showing() ) {
+
+		// Only active if the admin bar is showing.
+		if ( ! is_admin_bar_showing() ) {
 			return false;
 		}
 
-		// Gets post object. On front area we need to use get_queried_object to get the current post object.
-		if ( $this->is_admin_post_screen() ) {
-			$post = get_post();
-		} else {
-			$post = get_queried_object();
-		}
+		// Determine the queried object.
+		$queried_object = get_queried_object();
```

Comment:
Why was this changed? We should maintain the correct lookup for a single post also in the admin (i.e. block editor).

Reply:
this was changed in anticipation of other objects we might handle such as a term, comment or user/author. I'll change back to handling only posts explicitly, then we can add other object types later.

Reply:
Reverted to previous logic in https://github.com/google/site-kit-wp/pull/543/commits/710095214646bbcf40d63e09f2953f7c7e9fb776